app/assets/stylesheets/frontend/print/_html-publication.scss

```diff
@@ -0,0 +1,21 @@
+.html-publications-show {
+  nav.in-page-navigation {
+    display: block!important;
```

Comment:
Just seen the comment in the commit, sorry. Do you think it would be hard to remove the previous use of `important`?